.circleci/config.yml

```diff
@@ -55,23 +55,23 @@ jobs:
       - checkout
       - *checkout_submodule
 
-      - *restore_npm_cache
-      - *npm_install
+      - *restore_node_modules_cache
+      - *yarn_install
       - save_cache:
-          key: *npm_cache_key
+          key: *node_modules_cache_key
           paths:
             - node_modules
 
       - run:
           name: Run tests
-          command: npm test -- -- --coverage
+          command: yarn test --coverage
       - run:
           name: Upload code coverage reports to Codecov
           command: ./node_modules/.bin/codecov
 
       - run:
           name: Build KaTeX
-          command: npm run build
+          command: yarn run build
```

Comment:
This could be `yarn build`.

Reply:
@kevinbarabash I thought only certain commands can pass through, but it seems every commands can be pass through:

> It’s also possible to leave out the run in this command, each script can be executed with its name.
Note that built-in cli commands will have preference over your scripts, so you shouldn’t always rely on this shortcut in other scripts.

One place where our script may clash with built-in commands is `clean`, but Yarn won't be using it: https://github.com/yarnpkg/yarn/issues/2438.